src/main/webapp/components/query-manager/add-query.tsx

```diff
@@ -36,7 +35,7 @@ export default (props: AddQueryProps) => {
     <React.Fragment>
       <CreateSearch />
 
-      {!hasQueries && (
+      {showSecondarySection && (
```

Comment:
This prop feels a little awkward. Could we pull this div out of this component or pass it in as render props?